.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+bugs: ''

```

Comment:
Hi @harikanani, this would be the way to set the bug label here:

```suggestion
labels: ['bug']
assignees: []
```

Reply:
Ah, my bad, please recheck.